pages/1.11/security/ent/perms-reference/index.md

```diff
@@ -82,7 +82,7 @@ through Admin Router.
 | `dcos:adminrouter:licensing` <br> Controls access to the Licensing API.                                                                                                                                                                                                           | x    |   |   |   |   |
 | `dcos:adminrouter:package` <br> Controls access to the [Cosmos API](/1.11/api/master-routes/#cosmos), which provides access to the DC/OS Universe.                                                                                                                                | x    |   |   |   |   |
 | `dcos:adminrouter:secrets`<br> Controls access to [Secrets](/1.11/security/ent/secrets/) web interface and API.                                                                                                                                                                              | x    |   |   |   |   |
-| `dcos:adminrouter:service[:<group-name>]/<service-name>`<br> Controls access to the web interface and API of an installed DC/OS service.                                                                                                                                                        | x    |   |   |   |   |
+| `dcos:adminrouter:service:[<group-name>]/<service-name>`<br> Controls access to the web interface and API of an installed DC/OS service.                                                                                                                                                        | x    |   |   |   |   |
```

Comment:
If we want to make this more correct then the slash also needs to be in the squared brackets, no?

Reply:
These are valid examples:
```
dcos:adminrouter:service:org/dev/apps/svc1
```

```
dcos:adminrouter:service:svc1
```

Reply:
This is not a valid example:
```
dcos:adminrouter:service:/svc1
```

Reply:
True